app/controllers/flags_controller.rb

```diff
@@ -0,0 +1,74 @@
+class FlagsController < ApplicationController
+  before_action :set_flag, only: [:show, :edit, :update, :destroy]
+
+  # GET /flags
+  # GET /flags.json
+  def index
+    @flags = Flag.all
+  end
+
+  # GET /flags/1
+  # GET /flags/1.json
+  def show
+  end
+
+  # GET /flags/new
+  def new
+    @flag = Flag.new
+  end
+
+  # GET /flags/1/edit
+  def edit
+  end
+
+  # POST /flags
+  # POST /flags.json
+  def create
+    @flag = Flag.new(flag_params)
+
+    respond_to do |format|
+      if @flag.save
+        format.html { redirect_to @flag, notice: 'Flag was successfully created.' }
```

Comment:
Line is too long. [83/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.